data/query.go

```diff
@@ -0,0 +1,97 @@
+package data
+
+import (
+	"context"
+	"net/url"
+	"strconv"
+
+	"github.com/ONSdigital/log.go/log"
+)
+
+const (
+	// DefaultLimit - default values for limit query
+	DefaultLimit = 10
+	// DefaultSort - default values for sort query
+	DefaultSort = "relevance"
+	// DefaultPage - default values for page query
+	DefaultPage = 1
+)
+
+// GetLimitOptions returns all available limit options for search
+func GetLimitOptions() []int {
+	return []int{10, 25, 50}
+}
+
+// UpdateQueryWithOffset - removes page key and adds offset key to query to be then passed to dp-search-query
+func updateQueryWithOffset(ctx context.Context, query url.Values) url.Values {
+
+	page, err := strconv.Atoi(query.Get("page"))
+	if err != nil {
+		log.Event(ctx, "unable to convert search page to int - set to default "+strconv.Itoa(DefaultPage), log.INFO)
+		page = DefaultPage
+	}
+	limit, err := strconv.Atoi(query.Get("limit"))
+	if err != nil {
+		log.Event(ctx, "unable to convert search limit to int - set to default "+strconv.Itoa(DefaultLimit), log.INFO)
+		limit = DefaultLimit
+	}
+	offset := strconv.Itoa(((page - 1) * limit))
+	updateQuery := query
+	updateQuery.Set("offset", offset)
+	updateQuery.Del("page")
+	return updateQuery
+}
+
+// SetDefaultQueries ensures that all empty query fields are set to default
+func SetDefaultQueries(ctx context.Context, url *url.URL) *url.URL {
+	var found bool
+	query := url.Query()
+	pageQuery := query.Get("page")
+	if pageQuery == "" {
+		query.Set("page", strconv.Itoa(DefaultPage))
+	} else {
+		page, err := strconv.Atoi(pageQuery)
+		if err != nil {
+			log.Event(ctx, "unable to convert search page to int - default to page "+strconv.Itoa(DefaultPage), log.INFO)
+			query.Set("page", strconv.Itoa(DefaultPage))
+		} else {
+			if page < 1 {
+				log.Event(ctx, "page number is less than 1 - default to page "+strconv.Itoa(DefaultPage), log.INFO)
```

Comment:
Same here.

**Observation:** You are using `strconv.Itoa(DefaultPage)` a lot, to me this suggests that it might be beneficial to having a string var of this value
```
var defaultPageStr = "10"
```

Reply:
Agreed